core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalPortOwner.java

```diff
@@ -0,0 +1,28 @@
+package org.polarsys.capella.core.semantic.queries.basic.queries;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.emf.ecore.EObject;
+import org.polarsys.capella.common.helpers.query.IQuery;
+import org.polarsys.capella.core.data.cs.Component;
+import org.polarsys.capella.core.data.cs.PhysicalPort;
+
+/**
+ * return container of a @PhysicalPort
+ */
+public class PhysicalPortOwner implements IQuery {
+
+	@Override
```

Comment:
can you use the formatting of Capella 
https://github.com/eclipse/capella/wiki/How-to-contribute#code

Reply:
i applied the capella formatter on eclipse
but has no effect on this file